app/models/message_expectation.rb

```diff
@@ -0,0 +1,3 @@
+class MessageExpectation < ActiveRecord::Base
+  validates :content_id, uniqueness: true 
```

Comment:
A single piece of content may have multiple emails sent about it (one for each major update) so this isn't unique.